monkey/agent_plugins/credentials_collectors/mimikatz/config-schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "required": ["exclude_username_prefix"],
+    "properties": {
```

Comment:
We decided not to allow the user to select credential types?

Reply:
Nope, but we a tightly coupled to how Pypykatz is naming the credential types. See [this](https://github.com/skelsec/pypykatz/blob/1222aa210745ae66fa4a309f58c701186a4dcd8f/pypykatz/pypykatz.py#L348)